hack/common.sh

```diff
@@ -24,6 +41,7 @@ PYTHON_CLIENT_OUT_DIR=$OUT_DIR/client-python
 ARCHITECTURE="${BUILD_ARCH:-$(uname -m)}"
 HOST_ARCHITECTURE="$(uname -m)"
 KUBEVIRT_NO_BAZEL=${KUBEVIRT_NO_BAZEL:-false}
+export KUBEVIRT_CRI="$(determine_cri_bin)"
```

Comment:
Why does this need export but all the above variables do not?

Reply:
Thanks for catching that! It's now removed.
(I've also injected it into dockerized environment properly with docker/podman's `--env` flag.